webapp/custom-webpack.config.js

```diff
@@ -34,10 +34,6 @@ module.exports = {
       // openrosa-xpath-evaluator, but keeping it in adds 72.51KB to the bundle size.
       // https://github.com/medic/cht-core/issues/7324
       'node-forge': false,
-      // Only include the jquery version from the package.json (and not any different versions pulled in transitively).
-      // Once https://github.com/select2/select2/issues/5993 is resolved, we should try to coalesce back on one version
-      // of jquery and remove this alias.
-      'jquery': __dirname + '/node_modules/jquery',
```

Comment:
I have bumped us to use the same version of jquery as Enekto. Additionally, I suppose this _should_ be unnecessary with the `resolutions` section that we have in the package.json, right?